File: seed/ruby-sdk-v2/trace/lib/seed/submission/workspace_run_details.rb
# frozen_string_literal: true

module Seed
    module Types
        class WorkspaceRunDetails < Internal::Types::Model
            field :exception_v_2, Seed::Submission::ExceptionV2, optional: true, nullable: false
            field :exception, Seed::Submission::ExceptionInfo, optional: true, nullable: false
            field :stdout, String, optional: false, nullable: false

    end
end

File: seed/pydantic-v2/trace/baseline/resources/submission/types/workspace_run_details.py
from pydantic import BaseModel
from typing import Optional
from resources.submission.types.exception_v_2 import ExceptionV2
from resources.submission.types.exception_info import ExceptionInfo
from dt import datetime
from core.datetime_utils import serialize_datetime
class WorkspaceRunDetails(BaseModel):
    exception_v_2: Optional[ExceptionV2] = 
    exception: Optional[ExceptionInfo] = None
    stdout: str
    class Config:
        frozen = True
        smart_union = True
        json_encoders = {datetime: serialize_datetime}

